test/plug/adapters/cowboy_test.exs

```diff
@@ -108,6 +108,25 @@ defmodule Plug.Adapters.CowboyTest do
       assert on_response == my_onresponse
     end
 
+    defp my_onresponse_handler(_, _, _, req), do: res
+
+    test "elides the default onresponse handler if log_error_on_incomplete_requests is set to false and handles user-provided onresponse tuple " do
+      my_onresponse = {Plug.Adapters.CowboyTest, :my_onresponse_handler}
+      assert [Plug.Adapters.CowboyTest.HTTP,
+            _,
```

Comment:
Can you please make sure the indentation follows the same as the previous examples in this file?

Reply:
Ok, I'll check it.